spec/dummy/app/views/pages/index.html.erb

```diff
@@ -39,16 +22,50 @@
 
 <% cache @app_props_server_render do %>
   <% puts "=" * 80 %>
-  <% puts "server rendering react component" %>
+  <% puts "server rendering react components" %>
   <% puts "=" * 80 %>
   <!-- Default for prerender is true for the app, set in config/react_on_rails.rb -->
   <%= react_component("App", @app_props_server_render, trace: true) %>
+  <hr/>
+
+  <h1>Server Rendered/Cached React Component Without Redux</h1>
+  <code>
+    <%% cache @app_props_server_render do %><br/>
+    <%%    = react_component("HelloWorld", @app_props_server_render, trace: true) %><br/>
```

Comment:
[HelloWorldComponent](https://github.com/shakacode/react_on_rails/blob/master/spec/dummy/client/app/startup/ClientHelloWorldComponent.jsx) is a function that returns the component `<HelloWorld />`.
Here, the HelloWorld component is directly called without the wrapper function, and loaded server side (after exported by `ServerApp`), for demonstration purposes.
Below, the same HelloWorld component is loaded by the client.